app/build.gradle

```diff
@@ -65,6 +66,26 @@ android {
     androidExtensions {
         experimental = true
     }
+    splits {
+        abi {
+            enable gradle.startParameter.taskNames.contains("assembleRelease")
+            reset()
+            include 'arm64-v8a', 'armeabi', 'armeabi-v7a', 'mips', 'mips64', 'x86', 'x86_64'
+            universalApk false
```

Comment:
Make `universalApk true`, let's generate the Universal Apk also, it will be useful in some scenarios while sharing the APKs.